L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -132,6 +97,5 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
     # set the Stage 1 heavy ions-specific parameters
     process.simCaloStage1Digis.FirmwareVersion = cms.uint32(1)
     # move to the heavy ions draft L1 menu once the HLT has been updated accordingly
-    #process = L1Menu_CollisionsHeavyIons2015_v0(process)
-    process = L1Menu_Collisions2015_25ns_v2(process)
+    process = L1Menu_CollisionsHeavyIons2015_v0(process)
```

Comment:
Sorry for the delay - #8205.  I think that's all that's required but I'm checking it over as part of a general equivalency check.